flathub.json

```diff
@@ -0,0 +1,5 @@
+{
+    "only-arches": [
+        "x86_64"
```

Comment:
this is still utterly disappointing.

Reply:
*What* is disappointing..?

Reply:
that aarch64 users are left out.

Reply:
I checked up on it, and it looks like support *has* been added to Dart/Flutter.

However it would require the flatpak build process to actually support the Flutter build environment, or to build a script to setup the entire build environment. If and when there is something like the [github action](https://github.com/subosito/flutter-action) to enable the build environment for flatpak I'd be happy to enable builds for aarch64.